lms/templates/register-form.html

```diff
@@ -104,6 +104,12 @@ <h2 class="sr">${_('Required Information')}</h2>
       <label for="password">${_('Password')}</label>
       <input id="password" type="password" name="password" value="" required aria-required="true" />
```

Comment:
I know this is beyond the scope of this PR, but while we are in there, can we get rid of the ```aria-required="true"``` attr here on this line?  It is redundant and unnecessary.  The ```required``` attribute works for all user agents that support it, including screen readers.  explicitly setting this value for screen readers is unnecessary.